src/Spd.Manager.Licence/BizPortalUserManager.cs

```diff
@@ -0,0 +1,51 @@
+﻿using AutoMapper;
+using MediatR;
+using Spd.Manager.Shared;
+using Spd.Resource.Repository;
+using Spd.Resource.Repository.Biz;
+using Spd.Resource.Repository.PortalUser;
+using Spd.Resource.Repository.User;
+using Spd.Utilities.Shared.Exceptions;
+using System.Net;
+
+namespace Spd.Manager.Licence;
+internal class BizPortalUserManager
+    : IRequestHandler<BizPortalUserCreateCommand, BizPortalUserResponse>,
+    IBizPortalUserManager
+{
+    private readonly IMapper _mapper;
+    private readonly IPortalUserRepository _portalUserRepository;
+    private readonly IBizRepository _bizRepository;
+
+    public BizPortalUserManager(IMapper mapper, IPortalUserRepository portalUserRepository, IBizRepository bizRepository)
+    {
+        _mapper = mapper;
+        _portalUserRepository = portalUserRepository;
+        _bizRepository = bizRepository;
+    }
+
+    public async Task<BizPortalUserResponse> Handle(BizPortalUserCreateCommand request, CancellationToken ct)
+    {
+        PortalUserListResp existingUsersResult = await _portalUserRepository.QueryAsync(
+            new PortalUserQry() { OrgId = request.BizPortalUserCreateRequest.BizId, ContactRoleCode = ContactRoleCode.PrimaryBusinessManager },
```

Comment:
It should include two roles: PrimaryBusinessManager and BusinessManager

Reply:
Fixed.